app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+  "Total number of reports": "Número total de relatórios",
+  "Positive test results and close contact": "Resultados com teste positivo e contacto próximo",
+  "Clarifications data basis": "Esclarecimento sobre os dados",
+  "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:": "Perguntas sobre contacto próximo foram adicionadas às 22:30 de 16 de março. Alguns sintomas não fizeram parte do formulário desde o início:",
+  "Symptom": "Sintoma",
+  "Added to form": "Adicionado ao formulário",
+  "In total": "No total",
+  "people have reported that they experience symptoms.": "pessoas relataram que sentiram sintomas.",
+  "Number reported in COVID-19 infected patients": "Número relatado em pacientes infetados com COVID-19",
+  "Number reported": "Número relatado",
+  "Reports with symptoms": "Relatórios com sintomas",
+  "Reports": "Relatórios",
+  "In close contact with symptoms": "Em contacto próximo com sintomas",
+  "In close contact without symptoms": "Em contacto próximo sem sintomas",
+  "Close contact with someone who has tested positive": "Contacto próximo com alguém que testou positivo",
+  "Tested negative": "Negativos",
+  "Map": "Mapa",
+  "API": "API",
+  "I agree to set a cookie so I can change my health condition later e.g. to report recovery/getting sick": "Concordo em utilizar uma cookie para poder alterar o meu estado de saúde mais tarde, por exemplo para relatar situação de recuperação ou doença",
+  "required": "obrigatório",
+  "recommended": "recomendado",
+  "Below is your personal health information.": "Abaixo encontra-se a sua informação pessoal.",
+  "If you want to register another person on this computer, save or write down your personal link": "Se pretender registar outra pessoa neste computador, guarde ou anote o seu endereço pessoal",
+  "Your personal health information is cleared from this computer.": "A sua informação pessoal foi eliminada deste computador.",
+  "You can now use this computer for another user.": "Pode agora utilizar este computador para outra pessoa.",
+  "We stored your data in a cookie so come back later to update your status": "Guardámos os seus dados numa cookie, no seu computador, para que possa voltar mais tarde e atualizar o seu estado",
+  "If you use this computer with multiple users, please use this link to update your status.": "Se utilizar este computador com multiplos utilizadores, por favor utilize este endereço para atualizar o seu estado.",
+  "Write it down somewhere or save it on your computer.": "Anote-o ou guarde no seu computador.",
+  "The link has been copied.": "O endereço foi copiado.",
+  "clear this form": "limpe os dados deste formulário",
+  "https://who.int": "https://covid19.min-saude.pt/",
+  "To WHO.int": "Direção Geral de Saúde (DGS).",
+  "Are you from": "Vive em",
+  "Norway": "Norway",
+  "The Netherlands": "The Netherlands",
```

Comment:
I'm not sure if countries should be translated or not, but the translation is "**Países Baixos**"